src/wp-includes/fonts/class-wp-font-family.php

```diff
@@ -0,0 +1,615 @@
+	}
+
+	/**
+	 * Removes font family assets.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @return bool True if assets were removed, false otherwise.
+	 */
+	private function remove_font_family_assets() {
+		if ( $this->has_font_faces() ) {
+			foreach ( $this->data['fontFace'] as $font_face ) {
+				$were_assets_removed = $this->delete_font_face_assets( $font_face );
+				if ( false === $were_assets_removed ) {
+					return false;
+				}
+			}
+		}
+		return true;
+	}
+
+	/**
+	 * Removes a font family from the database and deletes its assets.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @return bool|WP_Error True if the font family was uninstalled, WP_Error otherwise.
+	 */
+	public function uninstall() {
+		$post = $this->get_data_from_post();
+		if ( null === $post ) {
+			return new WP_Error(
+				'font_family_not_found',
+				__( 'The font family could not be found.' )
+			);
+		}
+
+		if (
+			! $this->remove_font_family_assets() ||
+			! wp_delete_post( $post->ID, true )
+		) {
+			return new WP_Error(
+				'font_family_not_deleted',
+				__( 'The font family could not be deleted.' )
+			);
+		}
+
+		return true;
+	}
+
+	/**
+	 * Deletes a specified font asset file from the fonts directory.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param string $src The path of the font asset file to delete.
+	 * @return bool Whether the file was deleted.
+	 */
+	private static function delete_asset( $src ) {
+		$filename  = basename( $src );
+		$file_path = path_join( WP_Font_Library::get_fonts_dir(), $filename );
+
+		wp_delete_file( $file_path );
+
+		return ! file_exists( $file_path );
+	}
+
+	/**
+	 * Deletes all font face asset files associated with a given font face.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $font_face The font face array containing the 'src' attribute
+	 *                         with the file path(s) to be deleted.
+	 * @return bool True if delete was successful, otherwise false.
+	 */
+	private static function delete_font_face_assets( $font_face ) {
+		$sources = (array) $font_face['src'];
+		foreach ( $sources as $src ) {
+			$was_asset_removed = self::delete_asset( $src );
+			if ( ! $was_asset_removed ) {
+				// Bail if any of the assets could not be removed.
+				return false;
+			}
+		}
+		return true;
+	}
+
+
+	/**
+	 * Gets the overrides for the 'wp_handle_upload' function.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param string $filename The filename to be used for the uploaded file.
+	 * @return array The overrides for the 'wp_handle_upload' function.
+	 */
+	private function get_upload_overrides( $filename ) {
+		return array(
+			// Arbitrary string to avoid the is_uploaded_file() check applied
+			// when using 'wp_handle_upload'.
+			'action'                   => 'wp_handle_font_upload',
+			// Not testing a form submission.
+			'test_form'                => false,
+			// Seems mime type for files that are not images cannot be tested.
+			// See wp_check_filetype_and_ext().
```

Comment:
```suggestion
			/*
			 * Arbitrary string to avoid the is_uploaded_file() check applied
			 * when using 'wp_handle_upload'.
			 */
			'action'                   => 'wp_handle_font_upload',
			// Not testing a form submission.
			'test_form'                => false,
			/*
			 * Seems mime type for files that are not images cannot be tested.
			 * See wp_check_filetype_and_ext().
			 */
```
These should use the multiline comment format as specified in [the Inline Documentation standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments).

Reply:
Resolved by https://github.com/WordPress/wordpress-develop/pull/5285/commits/e948c1b764370bba2fa982b190e3c9dbb179cfab ✅